### PR TITLE
Feature/91 holo core update

### LINKED ISSUE #91: Upgrade Sensemaker to HDK 0.2.x
The older version of Holochain is now failing to compile on linux hosts due to problems with Lair Keystore.

### PR DESCRIPTION
**Pull request by pospi**
Resolves #91.

### CODE REVIEW

**Comment by adaburrows** [comment]
Interesting error: https://github.com/neighbour-hoods/sensemaker-lite/pull/92/checks#step:5:1771

**Comment by pospi** [comment]
Still haven't been able to figure out why that is happening. Can you check if you can execute locally?

`passed: 16  failed: 0  of 16 tests  (48.2s)`

I also want to revisit the base64 `EntryHash` encoding aspect to ensure nothing weird is happening with some of the BTreeHashMap return values.

**Comment by pospi** [comment]
Looks like we can confirm via @zippy that the expectation with any `HoloHash` is that `to_string()` will give you a Base64-encoded value.
https://discord.com/channels/727959104777224303/776952534873931806/844992352710950952

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -33,7 +33,18 @@ jobs:
...
         run: |
           cd $GITHUB_WORKSPACE
-          nix develop --command bash -c "pnpm i && pnpm test"
\ No newline at end of file
+          nix develop --command bash -c "pnpm i && echo Completed"
+
+      - name: debug-tsc-version
+        run: |
+          pushd tests/
+          pnpm ls typescript
```
Comment (adaburrows) [inline .github/workflows/test.yml]: This won't ever work. You need to wrap it with the same `nix develop --command bash -c "pnpm ls typescript"` as the rest of them.

**Comment by pospi** [comment]
Switching TypeScript module loaders seems to have been the fix- no more TS errors in CI and the tests hang indefinitely at the end as they do in a local env. Ready to merge.

**Comment by adaburrows** [comment]
This is already merged in somehow. I forget which PR included it.

**Pull request closed without merging**